### DISCUSSION TITLE
"ansible-auditd/templates/60-neo23x0.rules.j2" not updated is that wanted?

### DISCUSSION THREAD

**Discussion by 8ear** [juju4/ansible-adduser#34] [General]
Hi,
I saw that the file `https://github.com/juju4/ansible-auditd/tree/main/templates)/60-neo23x0.rules.j2` is not updated for long time.
Is that because you want that in that way or only because it was not updated?

Perhaps it make sense to add an additional ansible task to update it to current version:
````ansible
- name: Add auditd config template
  ansible.builtin.get_url:
    url: https://raw.githubusercontent.com/Neo23x0/auditd/master/audit.rules
    dest: /etc/audit/rules.d/60-neo23x0.rules
    mode: '0440'
    backup: yes
````

Kind regards

**Comment by juju4** [comment]
This seems the wrong repo to open this issue, more https://github.com/juju4/ansible-auditd/issues.
I synced today.
It can't be a get url as role is aggregating other rules.
